extensions/keycloak-authorization/runtime/pom.xml

```
@@ -12,6 +12,7 @@
 
     <artifactId>quarkus-keycloak-authorization</artifactId>
     <name>Quarkus - Keycloak Authorization - Runtime</name>
+    <description>Policy enforcer which uses permissions managed by Keycloak to control access to protected resources</description>
```

```suggestion
    <description>Policy enforcer using Keycloak-managed permissions to control access to protected resources</description>
```
would be a bit shorter? WDYT?